src/afl-fuzz-stats.c

```diff
@@ -778,10 +778,29 @@ void show_stats_normal(afl_state_t *afl) {
   if (unlikely(!banner[0])) {
 
     char *si = "";
+    char *fuzzer_name;
```

Comment:
pls make it a global in afl-fuzz.c and set it there, use extern here.

Reply:
Sorry... I did not got the idea...

I do not understand why we should do it. The variable is local, and is not needed outside of this scope...

Also I do not understand where is the proper place for this variable in `afl-fuzz.c` I did not found similar variable there...

May be it should be `afl_state` structure from  `afl-fuzz.h`? But I still not sure it is good idea...

I am not familiar with AFL++ architecture at all. I guess I need some guidance and explanations here...